### PR TITLE
Shirts component

### PR DESCRIPTION
**Pull request by safaaamro**
make shirtsComponent

### CODE REVIEW

**Review by mhmdtshref — APPROVED** [review]
Good, need to be merged to add the comparison Table component.

File: src/components/Shirts/shirts.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React, { Component } from "react";
+import "./shirts.css";
+import shirtsList from  "../../shirtsList.json";
+import Shirt from "../Shirt/shirt";
+
+class Shirts extends Component {
+ getShirts(){
+   const renderedShirts = shirtsList.map((shirt)=>{
```
Comment (jema28) [inline src/components/Shirts/shirts.js]: Destructuring would be great here e.g. `({imgUrl, label, price})` to avoid repeating "shirt".

**Review by jema28 — APPROVED** [review]
A few improvements can be made but great job overall 😸

**Review by jema28 — APPROVED** [review]
👍

**Pull request merged**